src/src_cpp/src_sequential/Benders.cpp

```diff
@@ -70,12 +70,13 @@ void Benders::build_cut() {
 	}
 	else {
 		get_slave_cut(slave_cut_package, _map_slaves, _options, _data);
+		_data.slave_cost = 0;
+		for (auto pairSlavenameSlavecutdata_l : slave_cut_package)
+		{
+			_data.slave_cost += pairSlavenameSlavecutdata_l.second.first.second[SLAVE_COST];
+		}
 	}
-	_data.slave_cost = 0;
-	for (auto pairSlavenameSlavecutdata_l : slave_cut_package)
-	{
-		_data.slave_cost += pairSlavenameSlavecutdata_l.second.first.second[SLAVE_COST];
-	}
+	
```

Comment:
It seems that correction from this PR https://github.com/AntaresSimulatorTeam/antares-xpansion/pull/64 will be reverted with this commit.

Reply:
Yes, it affects the behavior of Benders with the option RAND_AGGREGATION. It is an option to solve only a small number of subproblems at each iteration of the Benders decomposition.
1. This option is never used. It was implemented for research purposes at the very beginning of the project, and will be deleted when a proper cleaning of the optimization part will be done.
2. I changed this when I corrected the bug related to the operationnal cost, but if an operationnal cost is computed with the option RAND_AGGREGATION, it would have no sense. So I prefer to revert this.